### DISCUSSION TITLE
Predictive mean calculation confusion

### DISCUSSION THREAD

**Discussion by InfProbSciX** [cornellius-gp/gpytorch#1820] [General]
I do not understand the calculation that happens on [this line](https://github.com/cornellius-gp/gpytorch/blob/master/gpytorch/variational/variational_strategy.py#L116):

```python
            L = self._cholesky_factor(induc_induc_covar)
        interp_term = L.inv_matmul(induc_data_covar.type(_linalg_dtype_cholesky.value())).to(full_inputs.dtype)

        # Compute the mean of q(f)
        # k_XZ K_ZZ^{-1/2} (m - K_ZZ^{-1/2} \mu_Z) + \mu_X
        predictive_mean = (interp_term.transpose(-1, -2) @ inducing_values.unsqueeze(-1)).squeeze(-1) + test_mean
```

Shouldn't the predictive mean be `k_XZ * k_ZZ^{-1} * inducing_values + \mu_X`?
`interp_term` seems to calculate `k_XZ K_ZZ^{-1/2}`, `inducing_values` are `variational_distribution.loc`, so it seems like the computation happening here is just `k_XZ * k_ZZ^{>> -1/2 <<} * inducing_values + \mu_X` (isn't there a factor of `K_ZZ^{-1/2}` missing?)

**Comment by InfProbSciX** [comment]
Looks like the same question as https://github.com/cornellius-gp/gpytorch/discussions/1761.

**Reply by wjmaddox** [reply]
Oops, yes it is. This is due to whitening in the variational strategy, see the more explained version there.
